src/main/java/com/mraof/minestuck/block/MSHangingSignBlock.java

```diff
@@ -0,0 +1,28 @@
+package com.mraof.minestuck.block;
+
+import com.mraof.minestuck.blockentity.MSHangingSignBlockEntity;
```

Comment:
remove unused import

Reply:
I might have fixed this but not pushed it, are you still seeing an unused import?